### ISSUE TITLE
Crash when taking off from undefined celestial object

### ISSUE THREAD

**Issue by ChampTheLegend** [LemADEC/WarpDrive#351]
```
---- Minecraft Crash Report ----
// Don't do that.

Time: 7/31/18 6:28 AM
Description: Exception in server tick loop

java.lang.NullPointerException: Exception in server tick loop
	at cr0s.warpdrive.event.JumpSequencer.computeTargetWorld(JumpSequencer.java:913)
	at cr0s.warpdrive.event.JumpSequencer.state_getInitialVector(JumpSequencer.java:579)
	at cr0s.warpdrive.event.JumpSequencer.onUpdate(JumpSequencer.java:270)
	at cr0s.warpdrive.event.AbstractSequencer.updateTick(AbstractSequencer.java:24)
	at cr0s.warpdrive.event.WorldHandler.onServerTick(WorldHandler.java:119)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_478_WorldHandler_onServerTick_ServerTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:252)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:894)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:693)
	at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Thermos Version: cyberdynecc:Thermos:1.7.10-1614.58
	Plugins: CoreProtect, NoHunger, VoteRoulette, WorldEdit, ColoredSigns, TNTArrows, Skript, Offline_Teleporter, GroupManager, PlaceholderAPI, BungeeChatAPI, PlugMan, SecondPrefix, SkQuery, RandomTP, CosmicVaults, Votifier, WorldGuard, Multiverse-Core, ProtocolLib, mcore, Skellett, React, BanItem, skUtilities, PermaPotions, Vault, StaffBuddyAddon, Essentials, SharpSK, AutoAnnouncer, ObsidianAuctions, Trading, HolographicDisplays, WhatIsIt, KitAdder, skRayFall, DiscordSRV, ChestCommands, EssentialsChat, EssentialsSpawn, MassiveCore, EnjinMinecraftPlugin, CratesPlus, SignShop, Factions, ChatControl
	Disabled Plugins: 
	Operating System: Linux (amd64) version 4.4.0-131-generic
	Java Version: 1.8.0_171, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1907287688 bytes (1818 MB) / 3804758016 bytes (3628 MB) up to 9544663040 bytes (9102 MB)
	JVM Flags: 2 total; -Xms128M -Xmx10240M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 97
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 74 mods loaded, 63 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (server.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (server.jar) 
	UCHIJAAAA	kimagine{0.2} [KImagine] (minecraft.jar) 
	UCHIJAAAA	appliedenergistics2-core{rv2-stable-10} [AppliedEnergistics2 Core] (minecraft.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCHIJAAAA	OpenModsCore{0.10} [OpenModsCore] (minecraft.jar) 
	UCHIJAAAA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJAAAA	WarpDriveCore{1.7.10-1.4.2.-} [WarpDriveCore] (minecraft.jar) 
	UCHIJAAAA	securitycraft{v1.8.6} [SecurityCraft] ([1.7.10] SecurityCraft v1.8.6.jar) 
	UCHIJAAAA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJAAAA	BuildCraft|Core{7.1.23} [BuildCraft] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Energy{7.1.23} [BC Energy] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	ActuallyAdditions{1.7.10-r21} [Actually Additions] (ActuallyAdditions-1.7.10-r21.jar) 
	UCHIJAAAA	lightsabers{1.1.1} [Advanced Lightsabers Mod] (AdvancedLightsabers-1.7.10-1.1.1.jar) 
	UCHIJAAAA	appliedenergistics2{rv2-stable-10} [Applied Energistics 2] (appliedenergistics2-rv2-stable-10.jar) 
	UCHIJAAAA	armourersWorkshop{1.7.10-0.48.2} [Armourer's Workshop] (Armourers-Workshop-1.7.10-0.48.2.jar) 
	UCHIJAAAA	ThermalFoundation{1.7.10R1.2.6} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.6-118.jar) 
	UCHIJAAAA	ThermalExpansion{1.7.10R4.1.5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.1.5-248.jar) 
	UCHIJAAAA	BigReactors{0.4.3A} [Big Reactors] (BigReactors-0.4.3A.jar) 
	UCHIJAAAA	BiomesOPlenty{2.1.0} [Biomes O' Plenty] (BiomesOPlenty-1.7.10-2.1.0.1889-universal.jar) 
	UCHIJAAAA	BuildCraft|Transport{7.1.23} [BC Transport] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Factory{7.1.23} [BC Factory] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Silicon{7.1.23} [BC Silicon] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Robotics{7.1.23} [BC Robotics] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Builders{7.1.23} [BC Builders] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	ComputerCraft{1.75} [ComputerCraft] (ComputerCraft1.75.jar) 
	UCHIJAAAA	customnpcs{1.7.10d} [CustomNpcs] (CustomNPCs_1.7.10d(29oct17).jar) 
	UCHIJAAAA	PTRModelLib{1.0.0} [PTRModelLib] (Decocraft-2.4.2_1.7.10.jar) 
	UCHIJAAAA	props{2.4.2} [Decocraft] (Decocraft-2.4.2_1.7.10.jar) 
	UCHIJAAAA	endercore{1.7.10-0.2.0.39_beta} [EnderCore] (EnderCore-1.7.10-0.2.0.39_beta.jar) 
	UCHIJAAAA	MineFactoryReloaded{1.7.10R2.8.1} [MineFactory Reloaded] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	EnderIO{1.7.10-2.3.0.429_beta} [Ender IO] (EnderIO-1.7.10-2.3.0.429_beta.jar) 
	UCHIJAAAA	ForgeMultipart{1.2.0.345} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UCHIJAAAA	ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.12.jar) 
	UCHIJAAAA	IC2{2.2.827-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.827-experimental.jar) 
	UCHIJAAAA	GalacticraftCore{3.0.12} [Galacticraft Core] (GalacticraftCore-1.7-3.0.12.504.jar) 
	UCHIJAAAA	GalacticraftMars{3.0.12} [Galacticraft Planets] (Galacticraft-Planets-1.7-3.0.12.504.jar) 
	UCHIJAAAA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCHIJAAAA	ironbackpacks{1.7.10-1.2.20} [Iron Backpacks] (IronBackpacks-1.7.10-1.2.20.jar) 
	UCHIJAAAA	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) 
	UCHIJAAAA	mo{0.4.2} [Matter Overdrive] (MatterOverdrive-1.7.10-0.4.2.jar) 
	UCHIJAAAA	Mekanism{9.1.1} [Mekanism] (Mekanism-1.7.10-9.1.1.1031.jar) 
	UCHIJAAAA	MekanismGenerators{9.1.1} [MekanismGenerators] (MekanismGenerators-1.7.10-9.1.1.1031.jar) 
	UCHIJAAAA	MekanismTools{9.1.1} [MekanismTools] (MekanismTools-1.7.10-9.1.1.1031.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.1} [MFR Compat: Applied Energistics] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.1} [MFR Compat: BuildCraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.1} [MFR Compat: ForgeMicroblock] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatIC2{1.7.10R2.8.1} [MFR Compat: IC2] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.1} [MFR Compat: Thermal Expansion] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatVanilla{1.7.10R2.8.1} [MFR Compat: Vanilla] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UCHIJAAAA	OpenMods{0.10} [OpenMods] (OpenModsLib-1.7.10-0.10.jar) 
	UCHIJAAAA	OpenBlocks{1.6} [OpenBlocks] (OpenBlocks-1.7.10-1.6.jar) 
	UCHIJAAAA	openmodularturrets{2.2.11-245} [Open Modular Turrets] (OpenModularTurrets-1.7.10-2.2.11-245.jar) 
	UCHIJAAAA	harvestcraft{1.7.10j} [Pam's HarvestCraft] (Pam's HarvestCraft 1.7.10Lb.jar) 
	UCHIJAAAA	starwarsmod{1.3.3} [Parzi's Star Wars Mod] (Parzi's Star Wars Mod v1.3.3.jar) 
	UCHIJAAAA	StevesCarts{2.0.0.b18} [Steve's Carts 2] (StevesCarts2.0.0.b18.jar) 
	UCHIJAAAA	ThermalDynamics{1.7.10R1.2.1} [Thermal Dynamics] (ThermalDynamics-[1.7.10]1.2.1-172.jar) 
	UCHIJAAAA	WarpDrive{1.7.10-1.4.2.-} [WarpDrive] (WarpDrive-1.7.10-1.4.2.-.jar) 
	UCHIJAAAA	McMultipart{1.2.0.345} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UCHIJAAAA	ForgeMicroblock{1.2.0.345} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UD	MineFactoryReloaded|CompatAtum{1.7.10R2.8.1} [MFR Compat: Atum] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatBackTools{1.7.10R2.8.1} [MFR Compat: BackTools] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatChococraft{1.7.10R2.8.1} [MFR Compat: Chococraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.1} [MFR Compat: ExtraBiomes] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatForestry{1.7.10R2.8.1} [MFR Compat: Forestry] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatProjRed{1.7.10R2.8.1} [MFR Compat ProjectRed] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.1} [MFR Compat: Railcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.1} [MFR Compat: Sufficient Biomes] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.1} [MFR Compat: Thaumcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatTConstruct{1.7.10R2.8.1} [MFR Compat: Tinkers' Construct] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	UD	MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.1} [MFR Compat: TwilightForest] (MineFactoryReloaded-[1.7.10]2.8.1-174.jar) 
	OpenModsLib class transformers: [stencil_patches:ENABLED],[movement_callback:ENABLED],[player_damage_hook:ACTIVATED],[map_gen_fix:FINISHED],[gl_capabilities_hook:ENABLED],[player_render_hook:ENABLED]
	Class transformer null safety: all safe
	CoFHCore: -[1.7.10]3.1.4-329
	AE2 Version: stable rv2-stable-10 for Forge 10.13.2.1291
	ThermalFoundation: -[1.7.10]1.2.6-118
	ThermalExpansion: -[1.7.10]4.1.5-248
	MineFactoryReloaded: -[1.7.10]2.8.1-174
	ThermalDynamics: -[1.7.10]1.2.1-172
	EnderIO: Found the following problem(s) with your installation:
                  * The RF API that is being used (1.7.10R1.3.1 from <unknown>) differes from that that is reported as being loaded (1.7.10R1.0.13 from EnderIO-1.7.10-2.3.0.429_beta.jar).
                    It is a supported version, but that difference may lead to problems.
                 An unsupported base software is installed: 'thermos, cauldron, craftbukkit, mcpc, kcauldron'. This is NOT supported.
                 This may have caused the error. Try reproducing the crash WITHOUT this/these mod(s) before reporting it.
	AE2 Integration: IC2:ON, RotaryCraft:OFF, RC:OFF, BC:ON, RF:ON, RFItem:ON, MFR:ON, DSU:ON, FZ:OFF, FMP:ON, RB:OFF, CLApi:OFF, Waila:OFF, Mekanism:ON, ImmibisMicroblocks:OFF, BetterStorage:OFF
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 5 / 50; [GCEntityPlayerMP['ryaynb7'/390, l='build', x=102.70, y=236.00, z=1366.30](ryaynb7 at 102.69999998807907,236.0,1366.300000011921), GCEntityPlayerMP['_Anime'/2786, l='build', x=101.84, y=236.00, z=1371.59](_Anime at 101.84413705849599,236.0,1371.5934057626873), GCEntityPlayerMP['cerditopequeno'/6819, l='world', x=800.29, y=159.00, z=856.84](cerditopequeno at 800.2903157227945,159.0,856.8424072692254), GCEntityPlayerMP['UberNovaGamer'/8715, l='world', x=-4163.95, y=44.00, z=3260.30](UberNovaGamer at -4163.953230712859,44.0,3260.300000011921), GCEntityPlayerMP['Boften'/71405, l='world', x=787.06, y=159.00, z=906.28](Boften at 787.0644111912125,159.0,906.2849844273272)]
	Is Modded: Definitely; Server brand changed to 'thermos,cauldron,craftbukkit,mcpc,kcauldron,fml,forge'
	Type: Dedicated Server (map_server.txt)
```

**Comment by LemADEC** [comment]
You're trying to take off from an undefined celestial object.
Update your celestialObjects-*.xml configuration to define all planets properly.

**Comment by Garr890354839** [comment]
How, then, do I properly define it?

Here is a definition I have for a pre-existing dimension:
`                        <celestialObject id="Proxima">
				<parent>
					<center x="1000000" z="0" />
				</parent>
				<size x="15000" z="15000" />
				<dimension id="6789" isBreathable="true" gravity="normal">
					<center x="0" z="0" />
				</dimension>
				<render red="1.00" green="1.00" blue="1.00" alpha="1.00" texture="minecraft:textures/blocks/end_stone.png" periodU="150" />
				<render red="0.50" green="0.50" blue="0.40" alpha="0.30" />
			</celestialObject>`
And it crashed...
`java.lang.RuntimeException: Invalid celestial objects definition: update your configuration to fix this validation error, search your logs for 'CelestiaObjects validation error' to get more details.

    at cr0s.warpdrive.data.CelestialObjectManager.rebuildAndValidate(CelestialObjectManager.java:323)

    at cr0s.warpdrive.data.CelestialObjectManager.load(CelestialObjectManager.java:135)

    at cr0s.warpdrive.config.WarpDriveConfig.onFMLpreInitialization(WarpDriveConfig.java:554)

    at cr0s.warpdrive.WarpDrive.onFMLPreInitialization(WarpDrive.java:313)`
...

**Comment by LemADEC** [comment]
As the logs told you: _search your logs for 'CelestiaObjects validation error' to get more details._

**Comment by LemADEC** [comment]
No update for 4+ weeks, assuming fixed

**Comment by LemADEC** [comment]
Also, the crash is fixed in dev on 1.12.2 branch.
